Instabug-XCGLogger.podspec

```
@@ -21,9 +21,9 @@ Pod::Spec.new do |s|
     }
   s.author           = { "Instabug" => "contactus@instabug.com" }
   s.source           = { :git => "https://github.com/Instabug/Instabug-XCGLogger.git", :tag => "#{s.version}" }
-  s.platform         = :ios, '8.0'
+  s.platform         = :ios, '10.0'
   s.source_files     = 'Instabug-XCGLogger'
   s.requires_arc     = true
-  s.dependency 'Instabug', '>= 6.2'
-  s.dependency 'XCGLogger'
-end
\ No newline at end of file
+  s.dependency 'Instabug', '>= 8.0'
```

```suggestion
  s.dependency 'Instabug', '>= 10.0.3'
```
since the iOS version is 10.0